packages/dso-toolkit/src/components/headings/headings.scss

```diff
@@ -60,7 +62,9 @@ h5,
 .h5,
 h6,
 .h6 {
-  @include headings.margin-small();
+  &:not(.inline-heading) {
```

Comment:
Onze classnames zijn altijd prefixed met `dso-`.

Ik weet dat we nog het rijtje `.h1-.h6` hebben, maar die komen nog wel 'n keertje. Nieuwe classes altijd prefixen 👍